apps/client/src/features/control/playback/add-time/AddTime.tsx

```diff
@@ -28,9 +28,9 @@ export default function AddTime(props: AddTimeProps) {
   const handleAddTime = (direction: 'add' | 'remove') => {
     // API expects input in seconds
     if (direction === 'add') {
-      setPlayback.addTime(time / MILLIS_PER_SECOND);
+      setPlayback.addTime(time);
```

Comment:
suggestion: what do you think about renaming this variable `timeInMs` to avoid questions in future?